### PR TITLE
feat: capture errors

### PR DESCRIPTION
**Pull request by sxzz**
### Description

Capture errors.
image

### CODE REVIEW

**Comment by antfu** [comment]
For errors, maybe it would be better to disable compare as well as the diff? Maybe a dedicated view so the stacktrace could be clickable.

**Comment by sxzz** [comment]
Disabled diff and compare now. But I'm not interested in implementing clickable stack trace now. Maybe someone else can do it? 🤔

File: playground/src/App.vue [hunk 1]
```diff
@@ -7,3 +7,9 @@ import HelloWorld from './components/HelloWorld.vue'
 <template>
   <HelloWorld msg="Vite + Vue" />
 </template>
+
+<style lang="scss">
+.a {
+  color: red;
+}
```
Comment (antfu) [inline playground/src/App.vue:14]: What is this for?

Comment (sxzz) [reply playground/src/App.vue]: Oops, test code. Removed.

**Pull request merged**